.readthedocs.yml

```diff
@@ -32,11 +32,7 @@ python:
 # Let the build fail if there are any warnings
 sphinx:
     builder: html
-    fail_on_warning: true
-
-search:
-  ranking:
-    reference/apidoc/*: -7
+    fail_on_warning: false
```

Comment:
```suggestion
    fail_on_warning: false
    # Continue the RTD docs build even when it fails, we want to check the preview build to spot the error and see if it can be ignored (usually can't!). But, the reviewer should still go to the build to check if there are warnings in the build.
```